画面実装

suzukinan opened the pull request:
# WHAT
画面の実装

# WHY
メイン画面となるため

suzukinan committed:
test

Test

suzukinan committed:
画面の実装

View画面の実装

TOMARUMARU reviewed (commented):
BEMと&記法を用いるようにしてください

suzukinan committed:
修正

命名規則BEMを使用、&を使用
・index.html.haml
・style.scss

style.scss 221行
コメントアウトを削除

TOMARUMARU commented on app/controllers/messages_controller.rb:
```diff
@@ -0,0 +1,10 @@
+
+
+
+
```
最下行は改行一つで大丈夫です

suzukinan replied:
修正しました。

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットの使用は避けましょう

suzukinan replied:
修正しました。

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+    :css
+      /* Email styles need to be inline */
+  %body
+    = yield
```
mailer系のファイルは使用しないので削除して大丈夫です

suzukinan replied:
削除しました

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
mailer系のファイルは使用しないので削除して大丈夫です

suzukinan replied:
削除しました

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get 'messages' => 'messages#index'
```
root_pathに設定しましょう

suzukinan replied:
修正しました

suzukinan committed:
修正-2

messages_controller.rb
不要な改行を訂正しました。

application.html.haml
ハッシュの使用を避けました。

Mailer系の２ファイルの削除しました。

routes.rb
get ‘messages =>’ から
rootに変更しました。

TOMARUMARU reviewed (approved):
LGTM

suzukinan commented:
ありがとうございました！
